src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,19 @@
 import React from 'react';
+import { User } from '../../types/User';
+import './UserInfo.scss';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+type Props = {
+  user: User | null;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  return (
+    <>
+      {user ? (
```

Comment:
or even better to use [conditional rendering](https://react.dev/learn/conditional-rendering) - do not render this component at all if the user is not defined (handle inside PostInfo)